modules/broker-amqp/src/main/java/org/wso2/broker/amqp/codec/frames/ConnectionSecureOk.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.broker.amqp.codec.frames;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.channel.ChannelHandlerContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.broker.amqp.codec.BlockingTask;
+import org.wso2.broker.amqp.codec.handlers.AmqpConnectionHandler;
+import org.wso2.broker.common.data.types.LongString;
+import org.wso2.broker.common.data.types.ShortString;
+
+import javax.security.sasl.SaslException;
+import javax.security.sasl.SaslServer;
+
+/**
+ * AMQP connection.secure.ok frame.
```

Comment:
Shall we document the used parameters as well similar to other frames? E.g. https://github.com/wso2/message-broker/blob/master/modules/broker-amqp/src/main/java/org/wso2/broker/amqp/codec/frames/ConnectionOpen.java